pkg/cmd/pulumi/whoami.go

```diff
@@ -58,20 +59,50 @@ func newWhoAmICmd() *cobra.Command {
 			}
 
 			if verbose {
+				if jsonOut {
+					return outputWhoAmIJSON(WhoAmIJSON{
+						User:          name,
+						Organizations: orgs,
+						BackendURL:    b.URL(),
+					})
+				}
 				fmt.Printf("User: %s\n", name)
 				fmt.Printf("Organizations: %s\n", strings.Join(orgs, ", "))
 				fmt.Printf("Backend URL: %s\n", b.URL())
 			} else {
+				if jsonOut {
+					return outputWhoAmIJSON(WhoAmIJSON{
+						User: name,
+					})
```

Comment:
I'm not sure verbose should change which fields we put in the JSON object.
JSON is intended to be machine readable, so verbosity isn't really relevant.
I would put all three pieces in the output if jsonOut is set. So:

```
if jsonOut {
  return outputWhoAmiIJson(...)
}

if verbose {
  fmt.Printf("User: %s\n", ..)
  fmt.Printf("Organizations: %s\n", ..)
  fmt.Printf("Backend URL: %s\n", ..)
} else {
  fmt.Println(name)
}
```

Reply:
Good point, agree. Thank you, will do. 👍

Reply:
Pushed. ✅